api/filter/http/http_connection_manager.proto

```diff
@@ -105,7 +145,7 @@ message HttpConnectionManager {
   // requests. If not set, there is no idle timeout. When the idle timeout is
   // reached the connection will be closed. If the connection is an HTTP/2
   // connection a drain sequence will occur prior to closing the connection. See
-  // drain_timeout.
+  // *drain_timeout*.
```

Comment:
You can link to individual fields in the protodoc.py RST.

Reply:
Yeah was just being lazy. Was going to take another pass but will do it in this PR.